wolfcrypt/src/evp.c

```diff
@@ -2394,8 +2394,8 @@ int wolfSSL_EVP_DigestSignFinal(WOLFSSL_EVP_MD_CTX *ctx, unsigned char *sig,
             int nid = wolfSSL_EVP_MD_type(wolfSSL_EVP_MD_CTX_md(ctx));
             if (nid < 0)
                 break;
-            ret = wolfSSL_RSA_sign(nid, digest, hashLen, sig, &sigSz,
-                                   ctx->pctx->pkey->rsa);
+            ret = wolfSSL_RSA_sign_ex(nid, digest, hashLen, sig, &sigSz,
```

Comment:
What does "flag = 1" mean / do here?

Reply:
`flag = 1` means that `wolfSSL_RSA_sign_ex` only returns the value to which the return value of `*Verify*` should be compared to. In effect this returns the `wc_EncodeSignature` output for `RSA_PKCS1_PADDING` and the input for all other paddings.

Reply:
Thanks, please document this in the code or in the doxy